template/ig/privacy

```diff
@@ -2,19 +2,22 @@ Welcome to the W3C Privacy Interest Group, known as PING.
 
 We are delighted to have you join us. Our mission is to improve privacy on the Web.
 
-Our co-chairs are Pete Snyder (Brave), Christine Runnegar (invited expert), and Nick Doty (CDT). Philippe Le Hegaret is our W3C contact.
+Everyone who is involved in the {{ group.name }} is expected to behave in accordance with the W3C Code of Conduct:
+  https://www.w3.org/policies/code-of-conduct/
+
+Our co-chairs are Pete Snyder (Invited Expert), Christine Runnegar (invited expert), and Nick Doty (CDT). Philippe Le Hegaret is our W3C contact.
```

Comment:
for consistency, s/Invited Expert/invited expert/

Reply:
fixed. merci!